src/picongpu/include/particles/densityProfiles/FreeFormulaImpl.def

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2014-2017 Rene Widera
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+namespace picongpu
+{
+namespace densityProfiles
+{
+    template<typename T_ParamClass>
+    struct FreeFormulaImpl;
+}
```

Comment:
[https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#6-namespaces](url)    
But I can still see it ^^;